pom.xml

```diff
@@ -18,7 +17,15 @@
       <id>kohsuke</id>
       <name>Kohsuke Kawaguchi</name>
     </developer>
+      <developer>
+          <id>thawkins83</id>
+          <name>T.J. Hawkins</name>
+      </developer>
   </developers>
+    
+    <properties>
+        <jenkins.version>1.651</jenkins.version>
```

Comment:
It is preferable to use an LTS version. here you can stay on `1.625.3` or even older one